content/en/docs/concepts/security/rbac-good-practices.md

```diff
@@ -0,0 +1,101 @@
+
+### Workload Creation
+
+Users who are able to create workloads (e.g. pods) will be able to gain access to the underlying node if restrictions based on the [Pod Security Standards](/docs/concepts/security/pod-security-standards/) are not in place. Baseline or Restricted profiles should be used for all users with rights to create workloads in the cluster.
+
+Creating a workload in a namespace also grants indirect access to secrets in that namespace. Creating a pod in kube-system or a similarly privileged namespace can grant a user access to secrets they would not have through RBAC directly.  
+
+### Persistent Volume Creation
+
+As noted in the [Pod Security Policy](/docs/concepts/policy/pod-security-policy/#volumes-and-file-systems) documentation, access to create Persistent Volumes can allow for escalation of access to the underlying host.
+
+### Node/Proxy Access
+
+Users with access to the proxy sub-resource of node objects have rights to the Kubelet API, which allows for command execution on every pod on the node(s) which they have rights to. This access bypasses audit logging and admission control, so care should be taken before granting rights to this resource.
+
+### Escalate Right
+
+Generally the RBAC system prevents users from creating clusterroles with more rights than they possess. The exception to this is the escalate right. As noted in the [RBAC documentation](/docs/reference/access-authn-authz/rbac/#restrictions-on-role-creation-or-update) users with this right can effectively escalate their privileges.
+
+### Bind Right
+
+Similar to the Escalate right, granting users this right allows for bypass of Kubernetes in-built protections against privilege escalation, allowing users to create bindings to roles with rights they do not already have.
+
+### Impersonate Right
+
+This right allows users to impersonate and gain the rights of other users in the cluster. Care should be taken when granting it, to ensure that excessive permissions cannot be gained via one of the impersonated accounts.
+
+### CSRs and certificate issuing
+
+The CSR API allows for users with `create` rights to CSRs and `update` rights on `certificatesigningrequests/approval` where the signer is `kubernetes.io/kube-apiserver-client` to create new client certificates which allow users to authenticate to the cluster. Those client certificates can have arbitrary names including duplicates of Kubernetes system components. This will effectively allow for privilege escalation.
+
+### Token Request
+
+Users with `create` rights on `serviceaccounts/token` can create TokenRequests to issue tokens for existing service accounts. 
+
+### Control Admission Webhooks
+
+Users with control over `validatingwebhookconfigurations` or `mutatingwebhookconfigurations` can control webhooks that can read any object admitted to the cluster, and in the case of mutating webhooks, also mutate admitted objects.
```

Comment:
- I'd write ValidatingWebhookConfiguration and MutatingWebhookConfiguration in the text; however, you could also give an example of a dangerous RoleBinding (use a caution shortcode and inline comments) than mentions the API path fragment, eg `mutatingwebhookconfigurations`).
- if possible, [address the reader as you](https://kubernetes.io/docs/contribute/style/style-guide/#address-the-reader-as-you)